common/mercenaries/wc_mercenaries.txt

```diff
@@ -2,180 +2,169 @@
 # Also remember to tell the landed title to use this mercenary type instead.
 # Several titles can refer to the same type as well now.
 
+# IMPORTATN! Sum of units must be 1000
 brotherhood_of_the_horse_composition = {
 	levy_size = 1
-	heavy_infantry = 450
-	light_cavalry = 200
-	archers = 200
+	heavy_infantry = 600
+	light_cavalry = 350
 	elite_infantry = 50
 }
 order_of_the_silver_hand_composition = {
 	levy_size = 4
-	heavy_infantry = 500
-	light_cavalry = 100
-	archers = 150
-	elite_infantry = 100
-}
-scarlet_crusade_composition = {
-	levy_size = 4
-	archers = 250
-	heavy_infantry = 1100
-	light_cavalry = 225
+	heavy_infantry = 250
+	light_cavalry = 125
+	archers = 450
 	elite_infantry = 175
 }
-ymirjar_composition = {
+scarlet_crusade_composition = {
 	levy_size = 4
-	pikemen = 200
-	elite_infantry = 75
-	archers = 250
-	light_cavalry = 50
+	light_infantry = 100
+	heavy_infantry = 400
+	light_cavalry = 150
+	archers = 350
 }
 wardens_composition = {
-	levy_size = 1
-	heavy_infantry = 200
-	archers = 200
-	elite_infantry = 150
+	levy_size = 3
+	archers = 575
+	elite_infantry = 425
```

Comment:
Comp seems very squishy with 50%+ archers?

Reply:
`elite_infantry` are stronger then `archers`. If you consider the strength of units, then archers give 20%, while elite infantry give 80%.
https://docs.google.com/spreadsheets/d/1wLs_i8YQRL4_BmxPBdoibJQTC0vJmu8RbidPy7cwGZY/edit?usp=sharing